src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -300,4 +323,27 @@ export default class DropDownComponent extends ComponentBase<azdata.DropDownProp
 			'width': this.getWidth()
 		});
 	}
+
+	public get editableDropdownPlaceholder(): string {
```

Comment:
Any reason you're defaulting to an empty string instead of leaving it as undefined? Undefined would be better to use (clearer) for when the user specifically doesn't want a placeholder